services/QuillLMS/client/app/bundles/Teacher/components/classrooms/classroom_student_section.tsx

```diff
@@ -223,9 +226,15 @@ export default class ClassroomStudentSection extends React.Component<ClassroomSt
     this.setState({ selectedStudentIds: [] })
   }
 
-  handleClickViewAsStudentButton = () => this.props.viewAsStudent()
+  handleClickViewAsStudentButton = (e) => {
+    const { viewAsStudent } = this.props
+    viewAsStudent(e)
```

Comment:
I think passing `e` here is going to create issues since it looks like `viewAsStudent` is only expecting an argument that is an individual student's id.